### PR TITLE
Saving figures in temporary directory

### PR DESCRIPTION
**Pull request by shreyasbapat**
Solves #391 
Ran the notebook = "Plotting in 3D" with one change. 
screenshot from 2018-06-28 22-27-00
Changed this to : 
screenshot from 2018-06-28 22-29-43
As now, we don't allow to plot two bodies with different attractors in the same plot.

### CODE REVIEW

File: src/poliastro/tests/test_plotting2d.py [hunk 1]
```diff
@@ -92,7 +96,10 @@ def test_savefig_calls_prepare_plot():
     m = OrbitPlotter2D()
     earth = Orbit.from_body_ephem(Earth)
     m.plot(orbit=earth, label="Obj")
+    pwd = os.getcwd()
```
Comment (Juanlu001) [inline src/poliastro/tests/test_plotting2d.py]: Could you use

```
with tempfile.NamedTemporaryFile(...) as fp:
    m.savefig(filename=fp.name)
```

instead? I think it's much clearer

File: src/poliastro/tests/test_plotting3d.py [hunk 2]
```diff
@@ -106,7 +110,10 @@ def test_savefig_calls_prepare_plot():
     m = OrbitPlotter3D()
     earth = Orbit.from_body_ephem(Earth)
     m.plot(orbit=earth, label="Object")
+    pwd = os.getcwd()
```
Comment (Juanlu001) [inline src/poliastro/tests/test_plotting3d.py]: Same as above

File: src/poliastro/tests/test_plotting2d.py (hunk 1 above)
Comment (shreyasbapat) [reply src/poliastro/tests/test_plotting2d.py]: Oh yes! 
And sorry I didn't know that the cell was on purpose :sweat_smile:

File: src/poliastro/tests/test_plotting3d.py (hunk 2 above)
Comment (shreyasbapat) [reply src/poliastro/tests/test_plotting3d.py]: Ok

File: src/poliastro/tests/test_plotting2d.py (hunk 1 above)
Comment (shreyasbapat) [reply src/poliastro/tests/test_plotting2d.py]: Should I add a markup cell explaining the same thing? That the cell is on purpose? :sweat:

**Comment by Juanlu001** [comment]
Merged manually! Thanks @shreyasbapat :pray:

**Comment by shreyasbapat** [comment]
Sorry for the trouble!

**Pull request closed without merging**